daemon/protocol/python/scoot/daemon_cli.py

```diff
@@ -0,0 +1,156 @@
+"""
+Usage:
+  daemon_cli.py snapshot create <srcDir>
+  daemon_cli.py snapshot checkout <snapshotId> <destDir>
+  daemon_cli.py exec run <command> [--snapshotId=<sid>] [--timeout=<ns>]
+  daemon_cli.py exec poll <runIds> [--wait=<wait>] [--all]
```

Comment:
I think poll is weird; I'd suggest (though it may be for a different time) status